package.json

```diff
@@ -4818,6 +4823,11 @@
                 "type": "object",
                 "description": "Environment variables passed to dotnet. Only valid for hosted apps."
               },
+              "useVSDbg": {
+                "type": "boolean",
+                "default": false,
+                "description": "%generateOptionsSchema.useVSDbg.description%"
```

Comment:
>generateOptionsSchema [](http://example.com/codeflow?start=33&length=21)

Do you want this to be a launch configuration attribute, or just a preference? If you want it to be a launch configuration attribute, then you need modify the [launch-option generator](https://github.com/dotnet/vscode-csharp/blob/main/src/tools/generateOptionsSchema.ts) to add this, as I am almost certain when you re-run it, this change will get blown away. Also, I am guessing maybe you want a user-preference for this also, and that would happen automatically if you did that.

Reply:
I will remove it and keep only the useVSDbg option in the preferences.

![image](https://github.com/dotnet/vscode-csharp/assets/4503299/95698def-ea86-4d60-9717-63ba3848d91f)